scripts/build.mjs

```diff
@@ -108,13 +107,8 @@ async function main() {
   });
 
   // Remove the types file from the dist-src folder
-  const typeFiles = await glob([
-    "./pkg/dist-src/**/types.js.map",
```

Comment:
Is this file not being generated anymore?

Reply:
Nope, map files haven't been generated for the source files
https://github.com/octokit/webhooks.js/blob/main/scripts%2Fbuild.mjs#L106